egs/timit/ssr1/RESULTS.md

```diff
@@ -0,0 +1,35 @@
+- Environments
+  - date `Thursday June 4, 2020`
+  - operating system `Ubuntu 18.04.4 LTS (GNU/Linux 5.3.0-1020-gcp x86_64)`
+  - python version: `3.6.9 [GC 7.5.0]`
+  - espnet version: `espnet 0.7.0`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.0.1`
+
+
+- Results(`batch-size 16` `transformer-lr 20`using SpecAug)
+  ### CER
+
+  |dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+  |---|---|---|---|---|---|---|---|---|
+  |decode_test_decode_lm_word65000|100|5767|89.6|5.0|5.4|2.3|12.7|72.0
+  ### WER(removed sentences including abbreviations because there is no abbreviations in timit corpus)
```

Comment:
Maybe you can provide the number as it is?
Also, I extracted the WER numbers below from the PDF file in the ftp site. It seems that your result is quite good.
```
• Visual WSJ0 5k + Word-loop bigram (PCA features)
o Word level: 43.50%
o Phone level: 77.87%
• Visual WSJ0 5k + WSJ0 5k NVP bigram (PCA features)
o Word level: 79.08%
o Phone level: 90.21%
• Visual WSJ0 5k + WSJ0 5k NVP bigram (DCT features)
o Word level: 84.16%
o Phone level: 93.44%
```

Reply:
I think it is a limitation of the end-to-end system for this range of the amount of training data.
The HMM/DNN system was well designed to handle the issue.
We could do the same thing for the end-to-end system but we lose the simplicity, which we really want to avoid because this is the most important benefit.
So, I think the current number would be reasonable anyway as a simple recipe that everyone uses. We can do such additional efforts as a research item!

Reply:
Sorry for taking so long to fix this PR....
I believe now the recipe can work properly. But I don't know how to solve the `No module named 'numba.decorators'` problem in CI checking.. Could please tell me what to do next for merging this PR?